app/submitter/convert_payload_0_0_1.py

```diff
@@ -25,6 +30,7 @@ def convert_answers_to_payload_0_0_1(
     schema: QuestionnaireSchema,
     full_routing_path: Iterable[RoutingPath],
     progress_store: ProgressStore,
+    supplementary_data_store: SupplementaryDataStore,
```

Comment:
This is missing from the doc strings. There are also a few others.

Reply:
Added 👍 I couldn't actually find any others?

Reply:
Thought I saw more warning, maybe not.